README.md

```diff
@@ -5,8 +5,8 @@ This repository serves as a resting ground for all sorts of structure templates.
 Make sure to place the structure template prefab file in the [assets/prefabs/structures/](assets/prefabs/structures/) directory.
 Also add a picture of the structure to this [README](README.md) file.
 
-### Mountain Entrance
-![Mountain Entrance](https://github.com/Terasology/Mineshafts/raw/master/preview/MountainEntrance.jpg)
+### Glass Storage Room
+![Glass Storage Room](https://github.com/Torpedo99/AdditionalStructureTemplates/blob/master/preview/GlassStorageRoom.png)
```

Comment:
change the image paths to be relative in the repository so this path should be: preview/GlassStorageRoom.png